docs/examples/config.md

```diff
@@ -0,0 +1,13 @@
+# Product Configuration
+
+Generate example documentation
+
+
+
+::: examples/config/encoding-base-clingo.lp
+    handler: asp
+    options:
+        source: true
+        dependency_graph: true
+        glossary: true
+        predicate_table: true
```

Comment:
The encoding is not well documented (commented), I will take a look at that latter